### PR TITLE
Added HealthEquity

### PR DESCRIPTION
**Pull request by dimo414**

### CODE REVIEW

**Comment by Carlgo11** [comment]
@dimo414 
1.6KB:
healthequity 1

**Comment by dimo414** [comment]
Thanks, that saves me some time!

File: _data/investing.yml [hunk 1]
```diff
@@ -33,6 +33,12 @@ websites:
       exceptions:
           text: "2FA only available on investment accounts. Not available for 401K plans."
 
+    - name: HealthEquity
+      url: https://www.healthequity.com/
```
Comment (Carlgo11) [inline _data/investing.yml]: Is there a reason for me not being able to connect to the site? Neither with https nor http

Comment (dimo414) [inline _data/investing.yml]: Not sure, it works for me:

```
$ curl -I https://www.healthequity.com/
HTTP/1.1 302 Found
Cache-Control: private
Content-Length: 139
Content-Type: text/html; charset=utf-8
Location: /HE.aspx?ReturnUrl=%2f
Server: Microsoft-IIS/7.5
X-Powered-By: ASP.NET
Access-Control-Allow-Credentials: true
X-UA-Compatible: IE=edge
Date: Tue, 04 Aug 2015 02:54:14 GMT
Set-Cookie: ARRAffinity2=!su85XZCtyqaRx/486aLBj3Hq8XsL+GJgByW2bdJJvauMBF485Yiiah1gR8MiUlMjjRFecOfzgw8B20=;HttpOnly;Secure;Path=/;Version=1;
```

Slim possibility they restrict requests to US IP addresses?  That'd be screwed up, but then again they don't support tfa either, so.... :) Can you get to their investor site, http://ir.healthequity.com/ ?

Comment (Carlgo11) [inline _data/investing.yml]: Yeah http://ir.healthequity.com/ works but not www.healthequity.com. Odd, U.S. IP and all.

Comment (dimo414) [inline _data/investing.yml]: Very weird.  Is that a blocker for you?

Comment (RichJeanes) [inline _data/investing.yml]: I can get to https://www.healthequity.com from the US, but it's just a login page.  Interestingly enough, connecting over http gets you to their home page, the https gets you directly to the login page. So perhaps just switch to http?

**Comment by dimo414** [comment]
This was merged, but I don't see it on the site.  Just curious if there's a reason?

**Comment by jamcat22** [comment]
@dimo414 We review merges manually before merging `master` with `gh-pages`, so there is a delay between merges and the site being updated. Right now the delay is much longer than usual, because we're trying to fix some problems that updating Semantic-UI has caused.

You can read more about merges on [our wiki](https://github.com/2factorauth/twofactorauth/wiki/FAQ_Merge).

**Comment by fpigerre** [comment]
Hello @dimo414,
HealthEquity should now be visible on [twofactorauth.org](https://twofactorauth.org) :smile: 

Thanks again for the pull request,
psgs :palm_tree:

**Comment by dimo414** [comment]
Thanks @psgs!

**Pull request merged**